masterfabric-mobile/swift-camp
# CL - LazyVGrid Viper Implementation

## 📋 PR Description

This PR adds the implementation of a LazyVGridView following the VIPER architecture. A new button has been added to the HomeView, enabling navigation to the LazyVGridView screen. In the LazyVGridView items are created and loaded only when necessary.

---

## ✅ Checklist


---

## 🛠 Steps to Test

1. Launch the application.
2. Click the "LazyVGridView" button.
3. Ensure that LazyVGridView screen  appears
5. Scroll down - up to see new rectangles uploading while scrolling
6. Click the back button to verify that it navigates correctly to the HomeView.

---

## 🔗 Related Links

- Documentation: https://developer.apple.com/documentation/swiftui/lazyvgrid

---

### 📷 Screenshots (Optional)

![lazyVGridHome](https://github.com/user-attachments/assets/bc88c91f-09dc-4bb4-b191-18f098cec095)
![lazyVGridDetail](https://github.com/user-attachments/assets/08d2f329-244c-464c-aabe-abe5a8f91577)
